gerrymandering/gerrymandering.ipynb

```diff
@@ -46,22 +48,15 @@
    "outputs": [],
    "source": [
     "import holoviews as hv\n",
-    "from holoviews import opts\n",
+    "import hvplot.dask # noqa\n",
+    "import hvplot.pandas # noqa\n",
     "import geoviews as gv\n",
     "import datashader as ds\n",
     "import dask.dataframe as dd\n",
+    "import geopandas as gpd\n",
     "from cartopy import crs\n",
```

Comment:
Nitpick; convention is usually:
```suggestion
    "import cartopy.crs as ccrs\n",
```
https://scitools.org.uk/cartopy/docs/latest/gallery/vector_data/arrows.html#sphx-glr-gallery-vector-data-arrows-py